docs/APIs/tasks/tasks.md

```diff
@@ -1,3 +1,11 @@
 # Managing tasks
 
-The task management APIs handle creation and destruction of threads in mbed OS 5, as well as mechanisms for safe inter-thread communication. Threads are a core component of mbed OS 5 (even your `main` function starts in a thread of its own), so understanding how to work with them is a very important part of developing application for mbed OS 5. Pay particular attention to the [interrupt service routines](rtos.md#interrupt-service-routines) section, which contains important information about how the task management APIs can be used from an interrupt handler. 
+The task management APIs handle creation and destruction of threads in mbed OS 5, as well as mechanisms for safe interthread communication. Threads are a core component of mbed OS 5 (even your `main` function starts in a thread of its own), so understanding how to work with them is an important part of developing applications for mbed OS 5. Pay particular attention to the [interrupt service routines](rtos.md#interrupt-service-routines) section, which contains important information about how the task management APIs can be used from an interrupt handler. 
+
+* [RTOS](https://docs.mbed.com/docs/mbed-os-api-reference/en/5.3/APIs/tasks/rtos/): The CMSIS-RTOS core.
```

Comment:
This is all in the same project; you don't need the full URL, only the relative URL (which in this case is rtos.md)